src/app/shared/operators/state.ts

```diff
@@ -55,6 +56,6 @@ export function mapStateData<T, U>(dataMapper: (data: T) => U): OperatorFunction
  */
 export function mapErrorToState<T>(): OperatorFunction<FetchState<T>,FetchState<T>> {
   return pipe(
-    catchError(e => of(errorState(e instanceof Error ? e : new Error('unknown error')))),
+    catchError(e => of(errorState(e instanceof Error || e instanceof HttpErrorResponse ? e : new Error('unknown error')))),
```

Comment:
Isn't a change you forgot to remove?

Reply:
No this one was intentional, I can replace it by a predicate function that will test an implementation instead of instances, that might make actually more sense

Reply:
fixed